dll/win32/newdev/newdev.c

```diff
@@ -389,9 +403,8 @@ SearchDriverRecursive(
             {
```

Comment:
Two lines above (where I cannot comment), one could pass `wfd.cFileName` directly to the GetFileExt function, instead of the `FileName` buffer.

Reply:
Done